packages/compass-import-export/src/components/progress-bar/progress-bar.jsx

```diff
@@ -102,17 +107,23 @@ class ProgressBar extends PureComponent {
   };
 
   renderStats() {
-    const { docsTotal, docsWritten, progressLabel, progressTitle } = this.props;
+    const {
+      docsTotal,
+      docsWritten,
+      progressLabel,
+      progressTitle,
+      isAggregation,
+    } = this.props;
 
     const formattedWritten = formatNumber(docsWritten);
     const formattedTotal = formatNumber(docsTotal);
 
     return (
       <p
         className={style('status-stats')}
-        title={progressTitle(formattedWritten, formattedTotal)}
+        title={progressTitle(formattedWritten, formattedTotal, isAggregation)}
```

Comment:
Is this third param (`progressTitle`) extra here? The signature of `progressTitle` did not change anywhere